README.md

```diff
@@ -143,8 +143,8 @@ Catalyst-server ships with the following `shortstop` resolvers by default:
                 "cookieOptions": {
                     "isSecure": {
                         "$filter": "env.NODE_ENV",
-                        "development": false,
-                        "$default": true
+                        "production": true,
+                        "$default": false
```

Comment:
Checking against `production` instead since the absence of a value should imply development, not production.